ft_loop.py

```diff
@@ -0,0 +1,151 @@
+"""

```

Comment:
This is pretty cool! Maybe we add it the cli? I know I've written a similar script for myself a bunch of times. Your call here on if you want to do that though.